src/components/10-atoms/carousel/child.scss

```diff
@@ -1,6 +1,6 @@
 @mixin defaultAttributesChildElements {
   @include text(t4);
-  display: flex;
+  display: none !important;
```

Comment:
If that is needed, ok. If you just forgot it, please remove.

I'm writing this comment not because I do not trust you, but because this happened to me. :D